content/api/commands/and.md

```diff
@@ -68,8 +73,8 @@ Whatever was passed to the function is what is yielded.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>In most cases, `.and()` yields the same subject it was given from the
-previous command.</li></List>
+- In most cases, `.and()` yields the same subject it was given.
+- `.and()` is an assertion, and it is _safe_ to chain further methods.
```

Comment:
Are assertions commands or queries?

Reply:
Neither - they're assertions. They've always been a special case, and continue to be so.